CHANGELOG.md

```diff
@@ -4,6 +4,7 @@
 
 * [UIQM-641](https://issues.folio.org/browse/UIQM-641) Call `cleanBytesFields` function with correct arguments to fix 008 field.
 * [UIQM-640](https://issues.folio.org/browse/UIQM-640) Create authority | Make list of authority files in lookup alphabetical.
+* [UIQM-647](https://issues.folio.org/browse/UIQM-647) Import `useUserTenantPermissions` from `@folio/stripes/core`.
```

Comment:
Does this need to go into `8.0.1` bugfix, or can we move it to Ramsons?

Reply:
Ramsons for sure; this is not a bugfix. Do you want me to reopen this PR against a different branch? #master is the default branch and I sorta knew this shouldn't be targeted at 8.0.1 but it wasn't obvious what other branch would be a better target.

Reply:
No the branch is correct, just that it's under `8.0.1` section. We'll make sure not to release it for Q then